### PR TITLE
Fix branch names in “Start Contributing”

### PR DESCRIPTION
**Pull request by sujaypillai**
Replaced shortcode `{{< release-branch >}}` with `1.17`
Fixes [#19327](https://github.com/kubernetes/website/issues/19327)

### CODE REVIEW

File: content/en/docs/contribute/start.md [hunk 1]
```diff
@@ -209,16 +209,16 @@ to base your work on. Use these guidelines to make the decision:
   - Some localization teams work with a series of long-lived branches, and
     periodically merge these to `master`. This kind of branch has a name like
     dev-\<version>-\<language code>.\<team milestone>; for example:
-    `dev-{{< release-branch >}}-ja.1`.
+    `dev-1.17-ja.1`.
```
Comment (kbhawkey) [inline content/en/docs/contribute/start.md]: Hi @sujaypillai .
There is another shortcode that could work in this case:
```
{{< latest-semver >}}
```

Comment (kbhawkey) [reply content/en/docs/contribute/start.md]: Unless you are looking to add a different version number.

Comment (sujaypillai) [reply content/en/docs/contribute/start.md]: @kbhawkey - when i use the shortcode 
```
{{< latest-semver >}}
```
the output on line 212:
`dev-1.17 -ja.1`

I get an extra space in the string which instead should be `dev-1.17-ja.1`

Comment (kbhawkey) [reply content/en/docs/contribute/start.md]: 👀

Comment (kbhawkey) [reply content/en/docs/contribute/start.md]: OK. I created PR https://github.com/kubernetes/website/pull/19495.
Thanks for finding this issue!

Comment (sujaypillai) [reply content/en/docs/contribute/start.md]: I found the issue in shortcode template - [latest-semver.html](https://github.com/kubernetes/website/blob/b92ea5765e5bd0039ab0e2aac30d64400522fd31/layouts/shortcodes/latest-semver.html#L3)

`{{- $latestSemver }}` should be replaced by `{{- $latestSemver -}}`

Comment (sujaypillai) [reply content/en/docs/contribute/start.md]: Can I provide the fix here in the same pull request?

**Comment by kbhawkey** [comment]
@sujaypillai , The corrected shortcode should be available now.

File: content/en/docs/contribute/start.md [hunk 2]
```diff
@@ -209,16 +209,16 @@ to base your work on. Use these guidelines to make the decision:
...
   then you need to know the major and minor version of Kubernetes that
   the change will first appear in.
   - For example, if the feature gate JustAnExample is going to move from alpha
     to beta in the next minor version, you need to know what the next minor
     version number is.
   - Find the release branch named for that version. For example, features that
-    changed in the v{{< release-branch >}} release got documented in the branch
-    named `dev-{{< release-branch >}}`.
+    changed in the {{< latest-version >}} release got documented in the branch
+    named `dev-{{< latest-version >}}`.
```
Comment (sftim) [inline content/en/docs/contribute/start.md]:
```suggestion
    named `dev-{{< latest-semver >}}`.
```

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-19493--kubernetes-io-master-staging.netlify.com/docs/contribute/start/

**Comment by sftim** [comment]
/retitle Fix branch names in “Start Contributing”

**Comment by kbhawkey** [comment]
Thanks @sujaypillai .
/approve

**Pull request merged**